Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.Properties.cs

```diff
@@ -78,8 +78,9 @@ private static void OnBladeModeChanged(DependencyObject dependencyObject, Depend
 
                 if (bladeView.Items != null)
                 {
-                    foreach (BladeItem bladeItem in bladeView.Items)
+                    foreach (var item in bladeView.Items)
                     {
+                        var bladeItem = bladeView.GetBladeItem(item);
```

Comment:
We need to check for null

Reply:
Not possible. Every item either is a BladeItem or it's container is a BladeItem